include/s3select_json_parser.h

```diff
@@ -216,7 +216,7 @@ class JsonParserHandler : public rapidjson::BaseReaderHandler<rapidjson::UTF8<>,
 
     typedef enum {OBJECT_STATE,ARRAY_STATE} en_json_elm_state_t;
 
-    typedef std::pair<std::vector<std::string>, Valuesax> json_key_value_t;
+    typedef std::pair<std::string, Valuesax> json_key_value_t;
```

Comment:
?

Reply:
Used '/' to differentiate adjacent keys. get_key_path() returns that.

Reply:
It's not imp. Will make the test more readable